x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -208,20 +180,51 @@ describe('AddComment ', () => {
     const mockTimelineIntegration = { ...timelineIntegrationMock };
     mockTimelineIntegration.hooks.useInsertTimeline = useInsertTimelineMock;
 
-    const wrapper = mount(
-      <TestProviders>
-        <CasesTimelineIntegrationProvider timelineIntegration={mockTimelineIntegration}>
-          <AddComment {...addCommentProps} />
-        </CasesTimelineIntegrationProvider>
-      </TestProviders>
+    appMockRender.render(
+      <CasesTimelineIntegrationProvider timelineIntegration={mockTimelineIntegration}>
+        <AddComment {...addCommentProps} />
+      </CasesTimelineIntegrationProvider>
     );
 
     act(() => {
       attachTimeline('[title](url)');
     });
 
     await waitFor(() => {
-      expect(wrapper.find(`[data-test-subj="add-comment"] textarea`).text()).toBe('[title](url)');
+      expect(screen.getByTestId('euiMarkdownEditorTextArea')).toHaveTextContent('[title](url)');
+    });
+  });
+
+  describe('errors', () => {
```

Comment:
On these two tests maybe you could also guarantee that `Add comment` is disabled?

Reply:
Actually, `Add comment` is not disabled currently. So I replicated the existing behaviour of empty field error. 

However, since @mdefazio has provided advise on having common validation behaviour in all cases forms. I will create a separate PR for it.

Reply:
Ok so maybe I was thinking of `updating comment` but I could swear it worked like that when adding a new one too 🤔